.goreleaser-release-darwin.yaml

```diff
@@ -0,0 +1,58 @@
+before:
```

Comment:
What do you think about moving these release files out of the root? I think moving them into `.github` makes sense since they're exclusively for our GitHub CI.